spec/factories/user_factories.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+
+  factory :user do
```

Comment:
Add gem faker fro auto generation data for factories https://github.com/faker-ruby/faker